pkgs/applications/networking/browsers/ladybird/default.nix

```diff
@@ -47,17 +44,18 @@ stdenv.mkDerivation {
   ];
 
   cmakeFlags = [
-    "-DSERENITY_SOURCE_DIR=${serenity}"
     # Disable network operations
     "-DENABLE_TIME_ZONE_DATABASE_DOWNLOAD=false"
     "-DENABLE_UNICODE_DATABASE_DOWNLOAD=false"
   ];
 
-  # error: use of undeclared identifier 'aligned_alloc'
-  NIX_CFLAGS_COMPILE = toString (lib.optionals (stdenv.isDarwin && lib.versionOlder stdenv.targetPlatform.darwinSdkVersion "11.0") [
+  NIX_CFLAGS_COMPILE = [
+    "-Wno-error"
+  ] ++ lib.optionals (stdenv.isDarwin && lib.versionOlder stdenv.targetPlatform.darwinSdkVersion "11.0") [
```

Comment:
Why not use the new darwin sdk all the time?

Reply:
cc @wegank (authored those lines in #203532)

Reply:
This package is built with `llvmPackages_14.stdenv` in #203532, and combining different stdenvs on x86_64-darwin seems tricky.

Reply:
@SuperSandro2000 does that answer your question?